src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/BodyModelBinder.cs

```diff
@@ -54,11 +60,11 @@ public Task<ModelBindingResult> BindModelAsync(ModelBindingContext bindingContex
         /// <summary>
         /// Attempts to bind the model using formatters.
         /// </summary>
-        /// <param name="bindingContext">The <see cref="ModelBindingContext"/>.</param>
+        /// <param name="bindingContext">The <see cref="DefaultModelBindingContext"/>.</param>
```

Comment:
Not sure if the comment was made elsewhere but this is just the standard boilerplate so it would say `<see cref="ModelBindingContext"/>.`

However, this is a private method, so the docs can be deleted entirely, it's not really saying anything.